certified-connectors/IN-D Invoice Data Capture/apiDefinition.swagger.json

```diff
@@ -0,0 +1,473 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.1",
+    "title": "IN-D Invoice Data Capture",
+    "description": "A smart invoice processing solution that converts static documents (scanned, PDFs, screenshots, etc.) into usable information to fuel your business processes.IN-D Payables is an AI-led intelligent document classification and capture solution that automates the processing of all kinds of commercial documents used in Shipping, Logistics, and Manufacturing operations. Manual handling of documents slows the pace of business transactions."
+  },
+  "host": "payables-v3.in-d.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/e2e": {
+      "post": {
+        "summary": "InvoiceCapture",
+        "description": "Process invoices to extract the key attributes like invoice numbers, PO number, vendor name, etc., and export the data",
+        "operationId": "InvoiceCapture",
+        "parameters": [
+          {
+            "name": "Accept-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "v3.0",
+            "description": "Accept-Version"
+          },
+          {
+            "name": "Accept-Export-Type",
+            "in": "header",
+            "required": true,
+            "type": "integer",
+            "description": "Accept-Export-Type",
+            "format": "int64",
+            "default": 0
+          },
+          {
+            "name": "Accept-Attachment",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "True",
+            "description": "Accept-Attachment"
+          },
+          {
+            "name": "Accept-Route",
+            "in": "header",
+            "required": true,
+            "type": "integer",
+            "description": "Accept-Route",
```

Comment:
@gayathri35 descriptions for parameters should look like valid sentences too. You can refer to [an example from coding standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#definitions-and-parameters).